fix(distro-on-vms): minor tweaks to docs (#73)
sighupio/getting-started: distro-on-vms/README.md

- **Ansible** - used by furyctl to execute the roles from SD installers
- VMs OS: RHEL 8, RHEL 9, Rocky Linux 8, Rocky Linux 9, Debian 12, Alma Linux 9, Ubuntu 20, or Ubuntu 24

> - Valid FQDN for all the VMs, with a valid domain: for example, each VM should have a corresponding DNS entry like `worker1.example.tld`, `worker2.example.tld`, `master1.worker.tld`, etc.

Reviewer: ```suggestion
- Valid FQDN for all the VMs, with a valid domain: for example, each VM should have a corresponding DNS entry like `worker1.example.tld`, `worker2.example.tld`, `master1.example.tld`, etc.
```

- Fixed IP address for each VM.
- Two VMs for the load balancer Nodes (at least 1vCPU 1GB RAM each)
- An additional IP that will be used by keepalived to expose the two load balancers in HA, and a DNS record pointed to this IP for the control-plane address.

> - Three VMs for the master nodes (at least 2vCPU and 4GB RAM each)

Reviewer: ```suggestion
- Three VMs for the control-plane nodes (at least 2vCPU and 4GB RAM each)
```

- Three VMs for the worker nodes (at least 4vCPU and 8GB RAM each)
- `root` or passwordless sudo user SSH access to the VMs

> - Route53 credentials with permission to modify a zone (optional)

Reviewer: If we put this optional requirement here, we should also put somewhere in the guide the steps to configure it

DNS.2 = *.sighup.example.tld
```

> Change the values accordingly to your environment

Reviewer: ```suggestion
Change the values according to your environment
```

## Step 4 - Write the `furyctl.yaml` configuration file

> The next step is to write the cluster configuration file used by `furyctl`, in the tutorial directory is present a pre-compiled file that you can use as a starting point.

Reviewer: ```suggestion
The next step is to write the cluster configuration file used by `furyctl`, in the tutorial directory you can find a pre-compiled file that you can use as a starting point.
```

      additionalConfig: "{file://./haproxy-additional.cfg}"
```

Next we need to define the load-balancer nodes, each node will have a name and an IP address, additionally, we are also enabling keepalived on an additional floating (virtual) IP address, in this example `192.168.1.179`.

> [!IMPORTANT]
> Check which is the main interface that will be used for the keepalived IP, in this example the interface name is `enp0s8`.

We can give to the HAProxy statistics ("stats") page a username and a password, and we can also add additional custom configuration to the HAProxy running the load balancers. For example, we are using the additional configuration to load balance the ingress battery using the same load balancers as the control plane address.

> #### Kubernetes Master and Worker nodes

Reviewer: ```suggestion
#### Kubernetes Control-plane and Worker nodes
```

        taints: []
```

> Next we define the masters node and the worker nodes. The FQDN that will be used for each node will be the concatenation of the name and the `.spec.kubernetes.dnsZone` field.

Reviewer: ```suggestion
Next we define the control-plane nodes and the worker nodes. The FQDN that will be used for each node will be the concatenation of the name and the `.spec.kubernetes.dnsZone` field.
```

        type: calico
```

> In this piece of configuration, we are choosing to install Calico as CNI (Container Network Interface) in our cluster from the `module-networking` core module.

Author: ```suggestion
In this piece of configuration, we are choosing to install Calico as CNI (Container Network Interface) in our cluster from the `networking` core module.
```

This section configures the `module-logging` module. In this example we are installing Loki as log storage, and configuring the Logging operator with all the Flows and Outputs to send logs to the Loki stack.

> The minio configuration section is to deploy a minio cluster with the S3 bucket used by Loki to store logs, the storageSize selected defines the size for each minio disk, in total 6 disks split in 2 per 3 minio replicas.

Reviewer: ```suggestion
The minio configuration section is to deploy a MinIO cluster with the S3 bucket used by Loki to store logs, the storageSize selected defines the size for each MinIO disk, in total 6 disks split in 2 per 3 MinIO replicas.
```

        type: prometheus
```

> This section configures the `module-monitoring` module. The complete stack with Prometheus based on kube-prometheus and other monitoring tools.

Reviewer: ```suggestion
This section configures the `monitoring` module, which installs a complete monitoring stack with Prometheus based on kube-prometheus and other monitoring tools.
```

More about SD:

- [SD Documentation][docs]
- [SD Production-grade Installation][docs-prod-install]

<!-- Links -->
[distro-on-minikube]: https://github.com/sighupio/getting-started/tree/main/distro-on-minikube
[distro-on-eks]: https://github.com/sighupio/getting-started/tree/main/distro-on-eks
[docs]: https://docs.sighup.io

> [docs]: https://docs.sighup.io/docs/installation

Author: ```suggestion
[docs-prod-install]: https://docs.sighup.io/docs/installation
```
